### ISSUE TITLE
curl_share_setopt.3 misses availability information

### ISSUE THREAD

**Issue by m6w6** [curl/curl#826]
E.g. when did libcurl really support CURL_LOCK_DATA_SSL_SESSION?

**Comment by jay** [comment]
It was implemented starting in 7.23.0 which was released about 4 1/2 years ago. The others symbols are many years older. I think the issue is how far back do we want to go in the newer manuals. I could add something to that one though, how about:

~~~diff
diff --git a/docs/libcurl/curl_share_setopt.3 b/docs/libcurl/curl_share_setopt.3
index cbdbdc5..7242b0d 100644
--- a/docs/libcurl/curl_share_setopt.3
+++ b/docs/libcurl/curl_share_setopt.3
@@ -68,7 +68,7 @@ be used!
 SSL session IDs will be shared across the easy handles using this shared
 object. This will reduce the time spent in the SSL handshake when reconnecting
 to the same server. Note SSL session IDs are reused within the same easy handle
-by default.
+by default. Added in 7.23.0.
 .RE
 .IP CURLSHOPT_UNSHARE
 This option does the opposite of \fICURLSHOPT_SHARE\fP. It specifies that
~~~

**Comment by bagder** [comment]
I say commit that, as it at least answers @m6w6's question.

**Comment by m6w6** [comment]
Yeah, found it in the changelog. Usually I use the excellent symbol-in-version doc, but that didn't help, because the symbol existed way before implementation ;)

You know there are these RHELs, CentOSes and LTSes out there, which ship pretty old software and are even used for CI and production environments ;)

Thanks a lot!

**Comment by jay** [comment]
That's a good point Michael. @bagder would it be more appropriate then to say "Although this symbol was added in 7.10.3 it was not implemented until 7.23.0" or just keep it simple as "Added in 7.23.0"?

**Comment by bagder** [comment]
As this would've helped @m6w6 we can assume it might help someone else too, so I think a little more info there is better than a little less. I think.

**Comment by jay** [comment]
Thanks, landed in 694c2dc.
